modules/codegen/src/smithy4s/codegen/internals/SmithyToIR.scala

```diff
@@ -683,6 +683,28 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
 
     }
 
+  private def imputeZeroValuesOnDefaultTraits(
```

Comment:
Without this, all of the values in the Default Hint would be Document.null. If that were the case, then the codecs and such would need to be aware of what to do with a null default. Imputing the implied value seems preferable

Reply:
TIL 'impute'